ArcGISRuntimeSDKQt_CppSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,210 @@
+// [WriteFile Name=ContingentValues, Category=EditData]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import Esri.Samples 1.0
+
+Item {
+    id: sampleWindow
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: sampleWindow
+
+        MouseArea {
+            anchors.fill: view
+            visible: attributePrompt.visible
+            onClicked: mouse.accepted = attributePrompt.visible
+            onWheel: wheel.accepted = attributePrompt.visible
+        }
+
+        Component.onCompleted: {
+            // Set and keep the focus on SceneView to enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    // Declare the C++ instance which creates the scene etc. and supply the view
+    ContingentValuesSample {
+        id: contingentValuesSample
+        mapView: view
+    }
+
+    Control {
+        id: attributePrompt
+
+        // Expand the attributes pane depending on where the user clicks or taps
```

Comment:
The logic for the attributes pane might need to be reworked. Depending on where I click, the pane could be partly out of the view. But I cannot pan around to bring it back in view to complete the input fields. The only way I can interact with the map again is to hit "Discard". Below is an example. This is an issue in both C++ and QML

![popup_construded](https://user-images.githubusercontent.com/3746552/155246114-3ddf799c-cb6d-4127-abb5-9f5a4061d46a.PNG)